addons/web/static/src/core/notebook/notebook.scss

```diff
@@ -78,4 +77,12 @@
             }
         }
     }
+    @media only screen and (max-width: 767px) {
+        .o_notebook_content .oe-toolbar {
+            position: sticky;
```

Comment:
You can use our custom mixin
https://github.com/odoo/odoo/commit/5f1d32f3fb89f40740b25491caafb9a5710d555a#diff-3a53cced7e488eec86f2184a7fa68d1065cb1e0130689d0ebba047acd74c6774R30